docs/spec/v1beta3/providers.md

```diff
@@ -1474,6 +1474,34 @@ spec:
       name: flux-system
 ```
 
+##### Construction of Git Commit Status ID
```

Comment:
We should explain here what the commit status ID actually is and how it can be leveraged.

Reply:
Is it might be enough to link to GitHub and GitLab related API endpoints - as this documentation stays up2date? @makkes 

GitHub: https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28
GitLab: https://docs.gitlab.com/ee/api/commits.html#set-the-pipeline-status-of-a-commit